src/coreclr/jit/codegen.h

```diff
@@ -647,6 +647,7 @@ class CodeGen final : public CodeGenInterface
 
 #if defined(TARGET_AMD64)
     void genAmd64EmitterUnitTestsSse2();
+    void genAmd64EmitterUnitTestsApx();
```

Comment:
Not related to this PR really, but it'd be nice if we had similar tests for other ISAs/encodings (VEX, EVEX, etc). Sse2 itself is, afair, really just SimdLegacyEncoding.